eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/steps/processors/VariantProcessorTest.java

```diff
@@ -15,21 +31,24 @@
 
     @Before
     public void setUp() {
-        processor = new VariantProcessor("assembly","taxonomy",
-                "project");
+        processor = new VariantProcessor("assembly", 1111, "project");
```

Comment:
Maybe extracting these constants as attributes, as it seems to be intended that they always have the same values?